Remotion/Web/Core/UI/Controls/FormGridManager.cs

```diff
@@ -1194,26 +1191,9 @@ protected virtual void Dispatch (IDictionary values)
               Assertion.IsFalse (label is SmartLabel);
 #pragma warning restore 184
 
-              var textWithHotkey = HotkeyParser.Parse (label.Text);
-
               //  Label has associated control
-              if (!string.IsNullOrEmpty (label.AssociatedControlID))
-              {
-                ISmartControl? smartControl = control as ISmartControl;
-                if (smartControl != null && smartControl.UseLabel)
-                {
-                  label.Text = hotkeyFormatter.FormatText (textWithHotkey, false);
-                  label.AccessKey = hotkeyFormatter.FormatHotkey (textWithHotkey);
-                }
-                else
-                {
-                  label.Text = hotkeyFormatter.FormatText (textWithHotkey, false);
-                  label.AccessKey = hotkeyFormatter.FormatHotkey (textWithHotkey);
-                }
-              }
-              else
+              if (string.IsNullOrEmpty (label.AssociatedControlID))
               {
-                label.Text = textWithHotkey.Text;
                 label.AccessKey = "";
```

Comment:
This part goes away, too. The else was meant to handle the hotkey-scenario and we probably should document a breaking change for the AccessKey behavior but the whole section is no meaningless.